src/App.tsx

```diff
@@ -0,0 +1,34 @@
+import { useState } from 'react';
+
+import { Page } from './enums';
+// import { PageKeys } from './interfaces';
+import ForgotPassword from './pages/ForgotPassword';
+import Login from './pages/Login';
+import ResetPassword from './pages/ResetPassword';
+import SignUp from './pages/SignUp';
+
+function App() {
+  const [page, setPage] = useState('login');
```

Comment:
```suggestion
  const [currentPage, setCurrentPage] = useState<PageKeys>(PageKeys.login);
```

Then you don't need to use `as keyof PageKeys` on 22 line